sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+Changed API version to 2023-02-01.
```

Comment:
Would you like to add some tests?

Reply:
And please run `dotnet build /t:GenerateTests` to update the samples

Reply:
Oh thanks! I was running the command on a different csproj so wasn't getting any change. Ran it on the correct one now.